### PR TITLE
feat(ormb): update ormb Metrics

### PR DESCRIPTION
**Pull request by judgeeeeee**
**What type of PR is this?**
/kind feature
/area model

**What this PR does / why we need it**:
update ormb Metrics from `[]Metric` to `map[str][]Metric`
**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @FogDong @simon-cj 

**Does this PR introduce a user-facing change?**:

```release-note
update ormb Metrics from `[]Metric` to `map[str][]Metric`
```

### CODE REVIEW

**Comment by simon-cj** [comment]
LGTM

**Comment by songzhiwei7** [comment]
LGTM

File: pkg/model/types.go [hunk 1]
```diff
@@ -20,14 +20,14 @@ type Metadata struct {
...
-	Framework          string           `json:"framework,omitempty" yaml:"framework,omitempty"`
-	Metrics            []Metric         `json:"metrics,omitempty" yaml:"metrics,omitempty"`
-	HyperParameters    []HyperParameter `json:"hyperParameters,omitempty" yaml:"hyperParameters,omitempty"`
-	Signature          *Signature       `json:"signature,omitempty" yaml:"signature,omitempty"`
-	Training           *Training        `json:"training,omitempty" yaml:"training,omitempty"`
-	Dataset            *Dataset         `json:"dataset,omitempty" yaml:"dataset,omitempty"`
-	DirectoryStructure []string         `json:"directoryStructure,omitempty" yaml:"directoryStructure,omitempty"`
+	GPUType            string              `json:"gpuType,omitempty" yaml:"gpuType,omitempty"`
+	Framework          string              `json:"framework,omitempty" yaml:"framework,omitempty"`
+	Metrics            map[string][]Metric `json:"metrics,omitempty" yaml:"metrics,omitempty"`
```
Comment (FogDong) [inline pkg/model/types.go:25]: Can we change it to struct?

File: docs/docs_zh/introduction.md [hunk 2]
```diff
@@ -277,12 +277,14 @@ spec:
    "framework": "TensorFlow",
    "format": "SavedModel",
    "size": 9223372036854775807,
-   "metrics": [
-       {
-           "name": "acc",
-           "value": "0.928"
-       }
-   ],
+   "metrics":{
```
Comment (FogDong) [inline docs/docs_zh/introduction.md:280]:
```suggestion
   "metrics": { 
```
Same below

**Review by FogDong — COMMENTED** [review]
/lgtm

**Pull request merged**